pages/dashboard.tsx

```diff
@@ -0,0 +1,257 @@
+import React, { ReactElement, useState } from "react";
+import Head from "next/head";
+import {
+  Center,
+  Flex,
+  Tabs,
+  TabList,
+  TabPanels,
+  TabPanel,
+  Grid,
+  GridItem,
+  Table,
+  Thead,
+  Box,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from "@chakra-ui/react";
+import { TabPrimary } from "@/components/core/Tabs";
+import Sidebar from "@/components/sections/Sidebar";
+import { ButtonSecondary } from "@/components/core/Button";
+import DashboardHeader from "@/components/sections/DashboardHeader";
+import {
+  PropertyImage,
+  MultiWeightText,
+  PropertyDes,
+} from "@/components/sections/Listings";
+import { ListingType, testListing } from "../src/types";
+import { DashboardDisplay } from "../src/enum";
+
+interface ListingProps {
+  listings: ListingType[];
+}
+
+function DashboardPage(): ReactElement {
+  const [dashboardType, setDashboardType] = useState<DashboardDisplay>(
+    DashboardDisplay.Listings
+  );
+
+  const listingData = [
+    testListing,
+    testListing,
+    testListing,
+    testListing,
+    testListing,
+  ];
+
+  // const [listingData, setListingsData] = useState([
+  //   testListing,
+  //   testListing,
+  //   testListing,
+  //   testListing,
+  //   testListing,
+  // ]);
+
+  const openLandlordListing = () => {
+    setDashboardType(DashboardDisplay.Listings);
+  };
+
+  const openTenantListing = () => {
+    setDashboardType(DashboardDisplay.Tenants);
+  };
+
+  const openChatListing = () => {
+    setDashboardType(DashboardDisplay.Chat);
+  };
+
+  const TenantListings = ({ isOpen }: any) => {
+    return (
+      <Tabs w="100%" h="100%" isLazy display={isOpen ? "block" : "none"}>
+        <TabList>
+          <TabPrimary>Past</TabPrimary>
+          <TabPrimary>Current</TabPrimary>
+          <TabPrimary>Upcoming</TabPrimary>
+        </TabList>
+        <TabPanels
+          overflowY="scroll"
```

Comment:
Consider using `auto` instead of `scroll` if you don't want the scrollbar to show up when it doesn't need to

Reply:
Fixed! Also fixed it on the search page too